JustSaying/AwsTools/QueueCreation/SnsWriteConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Amazon.SimpleNotificationService.Model;
+
+namespace JustSaying.AwsTools.QueueCreation
+{
+    public class SnsWriteConfiguration
+    {
+        /// <summary>
+        /// Extension point allows custom error handling, including ability to specify whether exception has been explictly handled by consumer.
```

Comment:
Minor, but I find this comment a bit hard to understand - because it talks about consumers, but I think it means the consumer as in the caller (which is the publisher). And we also have consumers that are listening to messages, but they are a different type of consumer.

Reply:
That's a good point @tomhaigh, I will update.